packages/components/src/hooks/useWindowDimensions.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Returns the dimensions of the window.
```

Comment:
This is obvious from the name. If you want to provide documentation, specify which dimensions are returned (i.e., inner*) and why those are used instead of other choices.